src/handlers/opensea-api.js

```diff
@@ -15,11 +15,12 @@ const api = axios.create({
 
 export const apiGetAccountUniqueTokens = async (network, address, page) => {
   try {
-    const networkPrefix = network === NetworkTypes.mainnet ? '' : `${network}-`;
+    const isPolygon = network === NetworkTypes.polygon;
     const offset = page * UNIQUE_TOKENS_LIMIT_PER_PAGE;
-    const url = `https://${networkPrefix}api.opensea.io/api/v1/assets?exclude_currencies=true&owner=${address}&limit=${UNIQUE_TOKENS_LIMIT_PER_PAGE}&offset=${offset}`;
-    const data = await api.get(url);
-    return parseAccountUniqueTokens(data);
+    const url = `https://api.opensea.io/api/v1/assets?exclude_currencies=true&owner=${address}&limit=${UNIQUE_TOKENS_LIMIT_PER_PAGE}&offset=${offset}`;
+    const urlV2 = `https://api.opensea.io/api/v2/assets/matic?exclude_currencies=true&owner=${address}&limit=${UNIQUE_TOKENS_LIMIT_PER_PAGE}&offset=${offset}`;
+    const data = await api.get(isPolygon ? urlV2 : url);
```

Comment:
Since the only changing part is the path, this could be cleaned up a bit (not a blocker!)

Reply:
agreed, need to get with opensea and see what is currently available via v2 api and consolidate, wasnt able to get the mainnet response the same way as before